workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowRunner.kt

```diff
@@ -48,15 +49,11 @@ interface WorkflowRunner<out OutputT> {
   val renderings: StateFlow<Any>
 
   /**
-   * Returns the first (and only) [OutputT] value emitted by the workflow. Throws the cancellation
+   * Returns the next [OutputT] value emitted by the workflow. Throws the cancellation
    * exception if the workflow was cancelled before emitting.
    *
-   * The output of the root workflow is treated as a result code, handy for use
-   * as a sign that the host Activity or Fragment should be finished. Thus, once
-   * a value is emitted the workflow is ended and its output value is reported through
-   * this field.
    */
-  suspend fun awaitResult(): OutputT
+  suspend fun receiveOutput(): OutputT
```

Comment:
Should we expose the channel directly? I don't think so, we don't want to expose another cancellation API. Too bad there isn't a subset of `ReceiveChannel` that only exposes the select hooks without cancellation. And if we really need it, we can add later.